hathor/indexes/rocksdb_tx_group_index.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2021 Hathor Labs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from abc import abstractmethod
+from typing import TYPE_CHECKING, Iterable, Optional, Tuple, TypeVar
+
+from structlog import get_logger
+
+from hathor.indexes.rocksdb_utils import RocksDBIndexUtils
+from hathor.indexes.tx_group_index import TxGroupIndex
+from hathor.transaction import BaseTransaction
+
+if TYPE_CHECKING:  # pragma: no cover
+    import rocksdb
+
+logger = get_logger()
+
+KT = TypeVar('KT')
```

Comment:
I think that this:

```python
from collections.abc import Sized
KT = TypeVar('KT', Sized)
```

Could help solve a typing case (for when you need to do `key: KT; len(key)` for example.

Reply:
This was fixed with `KT = TypeVar('KT', bound=Sized)`. Thanks!